pinot-connectors/presto-pinot-driver/pom.xml

```diff
@@ -52,7 +53,8 @@
 
         <dependency>
             <groupId>org.apache.pinot</groupId>
-            <artifactId>pinot-spi</artifactId>
+            <artifactId>pinot-spi-jdk8</artifactId>
```

Comment:
Since the source code are the same, do we need to add these new modules?

Reply:
Yes, Presto side will pull and compile the dependencies transitively, and those jars have to be JDK 8 binary code.